core/local/local.go

```diff
@@ -105,9 +103,9 @@ func NewExecutionScheduler(
 	}
 
 	return &ExecutionScheduler{
-		runner:  runner,
-		logger:  logger,
-		options: options,
+		runner:       trs.Runner,
```

Comment:
Why leave this one? is it used a lot?

Reply:
probably just forgot to remove it, will do so now, along with `options`

Reply:
Resolved by https://github.com/grafana/k6/pull/2627/commits/ec45155e8a195a1dd3e97f2a154768125d33574a